misc/custom-script-shim

```diff
@@ -50,6 +50,34 @@ write_status() {
 	fi
 }
 
+check_binary_write_lock() {
+    set +e # disable exit on non-zero return code
+    local retry_attempts=0
+    while (( retry_attempts < 3 )); do
+        lsof_output="$(lsof ${bin})"
+        if [ "$?" -eq 0 ]; then
+            echo "${HANDLER_BIN} is open by the following processes: "
+            echo "${lsof_output}"
+            ((++retry_attempts))
+            echo "sleeping for 30 seconds before retry, attempt ${retry_attempts} of 3"
+            sleep 30
```

Comment:
Increase the maximum retry attempts and decrease the interval between them (10 attempts, 5 seconds between?)